public/app/features/alerting/alert_tab_ctrl.ts

```diff
@@ -87,6 +87,7 @@ export class AlertTabCtrl {
     switch (type) {
       case "email": return "fa fa-envelope";
       case "slack": return "fa fa-slack";
+      case "victorops": return "fa fa-victorops";
```

Comment:
Does Font Awesome have a VO icon? I don't see it in the [list](http://fontawesome.io/icons/).